outlines/generate/api.py

```diff
@@ -225,8 +242,9 @@ def __call__(
                 break
 
         generated_token_ids = self.get_generated_token_ids(
```

Comment:
Include `tokenizer.decode` in this function to make the code more understandable.